### DISCUSSION TITLE
optimizer and  lr_scheduler

### DISCUSSION THREAD

**Discussion by ZahraMESBAHI** [junyanz/pytorch-CycleGAN-and-pix2pix#1677] [Q&A]
Warning About lr_scheduler.step() Before optimizer.step(): The warning mentions that in PyTorch version 1.1.0 and later, you should call optimizer.step() before lr_scheduler.step(). The current order of calls in your code could lead to skipping the first learning rate update.
